### PR TITLE
Move credentials from environment variables to rails credentials 

### PR DESCRIPTION
**Pull request by aniket-k-kaushik**
## Notion card
https://saeloun.notion.site/Need-move-credentials-from-environment-variables-to-rails-credentials-edd145fbb9b24aa788f1c1e34c71d256

## Type of change
- [x] This change requires a documentation update

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [x] I have added automated tests for my code

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -7,13 +7,12 @@ default: &default
...
   <<: *default
-  url: <%= ENV['DATABASE_URL'].gsub('?', '_test?') %>
+  url: <%= Rails.application.credentials.database[:database_url] %>
 
 production:
   <<: *default
-  url: <%= ENV['DATABASE_URL'] %>
-  
\ No newline at end of file
+  url: <%= Rails.application.credentials.database[:database_url] %>
```
Comment (akhilgkrishnan) [inline config/database.yml]: Please revert these changes

Comment (aniket-k-kaushik) [reply config/database.yml]: done

**Comment by supriya3105** [comment]
@aniket-k-kaushik Can you resume work on this ticket after you finish the ones assigned to you currently.

**Comment by aniket-k-kaushik** [comment]
> @aniket-k-kaushik Can you resume work on this ticket after you finish the ones assigned to you currently.

@supriya3105 It was finished, the keys required for the credential files were only with Akhil and test are failing because, credential key aren't pushed with this PR. Rest of the things are done.

**Comment by aniket-k-kaushik** [comment]
@supriya3105  I'm closing this PR as @akhilgkrishnan informed he will be taking over the rest of the work and will create a separate PR
